### ISSUE TITLE
How to bump version without range type when in pre release

### ISSUE THREAD

**Issue by caohuilin** [atlassian/changesets#671]
## Affected Packages

@changesets/apply-release-plan

## Problem

When  I have a pre release in the monorepo, how I can bump dependencies version without range type.

Because my packages's user only install one package, but I want to install all dependencies that is pre version

**Comment by caohuilin** [comment]
example: https://github.com/caohuilin/monorepo-changeset-demo

**Comment by Andarist** [comment]
> when I use pre release to bump version, in @modernjs-test/module-2 packages will change @modernjs-test/module-1 to "workspace: ^0.1.1-canary.0", and publish will change to "^0.1.1-canary.0"

This is correct, right? Nothing wrong about it - or are you saying that this should be behave differently?

> user only install @modernjs-test/module-2 in his project. when want to use pre release version, the @modernjs-test/module-1 version will install 0.1.1, not 0.1.1-canary.0

Could you rephrase? From what I can tell if somebody installs `@modernjs-test/module-test-2@0.1.2-canary.0` the `@modernjs-test/module-test-1@0.1.2-canary.0` will be installed with it. And that is correct.

A separate note is that in this situation the published dependency range for `@modernjs-test/module-test-1` in `@modernjs-test/module-test-2` should lose the `^` modifier. Otherwise we are kinda risking invalid installs in the future as potentially incompatible `@modernjs-test/module-test-1@0.1.2-canary.1` (that would be created by a future release) could be installed instead of the correct `@modernjs-test/module-test-1@0.1.2-canary.0`. That conversion is lossy though and I haven't figured out how to best "bring it back" when exiting the pre mode
